swig/yui.i

```diff
@@ -269,8 +269,10 @@ class Exception;
   { return ($self != i); }
 #endif
 #if defined(SWIGPYTHON)
-  int __cmp__( YItem *i )
-  { return ($self - i); }
+  int __eq__( YItem *i )
+  { return ($self == i); }
+  int __ne__( YItem *i )
+  { return ($self != i); }
```

Comment:
This change is not needed if python 2 binding is used, but rebuilding all to use python 3 building does not work. The change works for both python 2 and python 3, that avoid different implementation for the same language even if in a different version.
You are not releasing anything for python3 atm, maybe we could check if the work done in our spec files to build python3 could be merged here as well... but that in another request of course :+1: